### PR TITLE
Changed events and twig extensions to use new class.

### PR DESCRIPTION
**Pull request by mickel1138**

### CODE REVIEW

File: src/Twig/Extensions.php [hunk 1]
```diff
@@ -37,7 +40,7 @@ class Extensions extends \Twig_Extension
      */
     public function __construct(KernelInterface $kernel, MaintenanceMode $maintenanceMode)
     {
-        $this->kernelRootDir = $kernel->getRootDir();
+        $this->kernelRootDir = $kernel->getProjectDir();
```
Comment (mickel1138) [inline src/Twig/Extensions.php]: Rootdir is not the same as ProjectDir, so dataland breaks.

Comment (fightingtexasaggie) [reply src/Twig/Extensions.php]: also new search is throwing a 500 now..looking into it

Comment (mickel1138) [reply src/Twig/Extensions.php]: Adding ` . '/src';` would give the same result.

Comment (fightingtexasaggie) [reply src/Twig/Extensions.php]: https://pastebin.com/fHjwy0ju  
Looks like our LogActionItemEvent isn't worknig as an Eventdispatcher Event.

Comment (mickel1138) [reply src/Twig/Extensions.php]: That's just a type hinting issue. It's parent must be updated too.

Comment (mickel1138) [reply src/Twig/Extensions.php]: It was more than that, the dispatch function signature changed.

File: src/Event/EntityEventDispatcher.php [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 namespace App\Event;
 
-use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
 
 use App\Entity\Entity;
```
Comment (mickel1138) [inline src/Event/EntityEventDispatcher.php]: Gotta switch the dispatch method around also on line #42

**Comment by mickel1138** [comment]
> Is src/Event/DatasetIndexSubscriber.php needing to be changed as well?

That stays in the Component namespace for now, as per Symfony instructions.

**Pull request merged**